### ISSUE TITLE
adding a map with a WMTS layer in a geostory fails

### ISSUE THREAD

**Issue by landryb** [georchestra/mapstore2-georchestra#484]
trying to create a geostory, adding a media of type 'map', importing a map.json where i have a WMTS background and two layers from the local geoserver.

importing the map.json works fine, but validating blows ms2.

same error with my default map which uses WMTS backgrounds.

using the OSM layer as background (without any other layers) doesnt blow so i guess this is related to WMTS layers.
```
[geOrchestra.js:2:6997131](https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9)
Uncaught TypeError: e is null
    l https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    y https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    _n https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    E https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    _ https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    i https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    z https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    c https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    c https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    default https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    v https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    onResize https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    v https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    E https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    _ https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    setTimeout handler*b https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    x https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    ro https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    ro https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    i https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    bi https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Da https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Fs https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    ol https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    nl https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    qs https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Ho https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    unstable_runWithPriority https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    zo https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Ho https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Zo https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Ks https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    wn https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
[geOrchestra.js:2:3113782](https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9)
TypeError: e.tileMatrixSet.filter is not a function
    A https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    M https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    i https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    E https://georchestra.dev.craig.fr/mapstore/dist/4385.b4bc8dcb691cbab88fb9.chunk.js:1
    componentDidMount https://georchestra.dev.craig.fr/mapstore/dist/4385.b4bc8dcb691cbab88fb9.chunk.js:1
    ll https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    unstable_runWithPriority https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    zo https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    sl https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    qs https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Ho https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    unstable_runWithPriority https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    zo https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Ho https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Zo https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    zs https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    enqueueSetState https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    setState https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    setPlugins https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    promise callback*setPlugins https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    componentDidMount https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    ll https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    unstable_runWithPriority https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    zo https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    sl https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    qs https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Ho https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    unstable_runWithPriority https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    zo https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Ho https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Zo https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Ks https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    wn https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
[geOrchestra.js:2:7852333](https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9)
TypeError: e.tileMatrixSet.filter is not a function
    A https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    M https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    i https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    E https://georchestra.dev.craig.fr/mapstore/dist/4385.b4bc8dcb691cbab88fb9.chunk.js:1
    componentDidMount https://georchestra.dev.craig.fr/mapstore/dist/4385.b4bc8dcb691cbab88fb9.chunk.js:1
    ll https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    unstable_runWithPriority https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    zo https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    sl https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    qs https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Ho https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    unstable_runWithPriority https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    zo https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Ho https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Zo https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    zs https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    enqueueSetState https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    setState https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    setPlugins https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    promise callback*setPlugins https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    componentDidMount https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    ll https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    unstable_runWithPriority https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    zo https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    sl https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    qs https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Ho https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    unstable_runWithPriority https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    zo https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Ho https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Zo https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Ks https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    wn https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
[geOrchestra.js:2:9864199](https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9)
TypeError: e.tileMatrixSet.filter is not a function
    A https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    M https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    i https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    E https://georchestra.dev.craig.fr/mapstore/dist/4385.b4bc8dcb691cbab88fb9.chunk.js:1
    componentDidMount https://georchestra.dev.craig.fr/mapstore/dist/4385.b4bc8dcb691cbab88fb9.chunk.js:1
    ll https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    unstable_runWithPriority https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    zo https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    sl https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    qs https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Ho https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    unstable_runWithPriority https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    zo https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Ho https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Zo https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    zs https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    enqueueSetState https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    setState https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    setPlugins https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    promise callback*setPlugins https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    componentDidMount https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    ll https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    unstable_runWithPriority https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    zo https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    sl https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    qs https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Ho https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    unstable_runWithPriority https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    zo https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Ho https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Zo https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Ks https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    wn https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
[geOrchestra.js:2:7852333](https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9)
TypeError: e.tileMatrixSet.filter is not a function
    A https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    M https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    i https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    E https://georchestra.dev.craig.fr/mapstore/dist/4385.b4bc8dcb691cbab88fb9.chunk.js:1
    componentDidMount https://georchestra.dev.craig.fr/mapstore/dist/4385.b4bc8dcb691cbab88fb9.chunk.js:1
    ll https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    unstable_runWithPriority https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    zo https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    sl https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    qs https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Ho https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    unstable_runWithPriority https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    zo https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Ho https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Zo https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    zs https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    enqueueSetState https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    setState https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    setPlugins https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    promise callback*setPlugins https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    componentDidMount https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    ll https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    unstable_runWithPriority https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    zo https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    sl https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    qs https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Ho https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    unstable_runWithPriority https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    zo https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Ho https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Zo https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    Ks https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
    wn https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9:2
[geOrchestra.js:2:9864199](https://georchestra.dev.craig.fr/mapstore/dist/geOrchestra.js?b4bc8dcb691cbab88fb9)
```

**Comment by tdipisa** [comment]
@landryb from what I understand the use case you are adopting is currently unsupported by the GeoStory Map Editor: GeoStory imports the map configuration incorrectly without assigning the tilematrix sets to the layer while using it's own internal map editor. If I remember well we have already seen the same in the past. Can you please create an issue on MS (using the bug template) by specifying the steps to reproduce and providing also the map.json you have used? You can assign the issue to me. Thank you

**Comment by landryb** [comment]
tested working fine with

MapStore2-geOrchestra	:[2022.01.xx](https://github.com/georchestra/mapstore2-georchestra/commits/2022.01.xx)[(3591c0d4)](https://github.com/georchestra/mapstore2-georchestra/commit/3591c0d4e62d80a53ce830a567d3813e3eb4a4b1)
MapStore2	:[2022.01.xx](https://github.com/geosolutions-it/MapStore2/commits/2022.01.xx)[(82446d14)](https://github.com/geosolutions-it/MapStore2/commit/82446d14d02fd6dff7c859a41fd7ee3927028a9a)

probably fixed by geosolutions-it/MapStore2#8028
